src/org/thoughtcrime/securesms/ConversationActivity.java

```diff
@@ -1676,19 +1676,22 @@ public void initializeContactRequest() {
       handleReturnToConversationList(extras);
     });
 
-    String question;
+    String question = null;
     if (dcChat.isMailingList()) {
       question = getString(R.string.ask_show_mailing_list, dcChat.getName());
     } else {
-      final int[] members = dcContext.getChatContacts(chatId);
-      if (members.length == 1) {
-        DcContact dcContact = dcContext.getContact(members[0]);
-        question = getString(R.string.ask_start_chat_with, dcContact.getNameNAddr());
-      } else {
-        question = "";
+      if (dcChat.getType() == DcChat.DC_CHAT_TYPE_GROUP) {
```

Comment:
maybe more readable to use `dcChat.isGroup()`

Reply:
Although `isGroup()` actually does `dcChat.getType() == DcChat.DC_CHAT_TYPE_GROUP || dcChat.getType() == DcChat.DC_CHAT_TYPE_MAILINGLIST`, which is rather a hack and I didn't want to add more usages. We should probably make a PR that renames `isGroup()` to `isMultiuser()`, and then later introduce a method `isGroup()` that checks what the name says.